styles.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: Inter;
+  src: url("assets/fonts/Inter-Bold.ttf");
+}
+
+@font-face {
+  font-family: Inter-Light;
+  src: url("assets/fonts/Inter-Light.ttf");
+}
+
+.heading {
+  display: flex;
+  align-self: center;
+  justify-content: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  line-height: 52px;
+  margin: 0;
+  padding-bottom: 5px;
+  font-weight: 700;
+  text-align: center;
+  width: 60vw;
+}
+
+.intro {
+  font-family: 'Inter-Light', sans-serif;
+  align-self: center;
+  text-align: center;
+  font-size: 20px;
+  margin: 0;
+  width: 60vw;
+  line-height: 28px;
+  font-weight: 100;
+}
```

Comment:
- [ ] I can see you're struggling to get the spacing of the texts correct as an example let's fix the paragraph issue I've noticed part of the problem is from the smaller width at line `31` you can increase it to around `80vw` to match the design.
- [ ] You can now work on the heading by applying the same principle. You can test out different properties by inspecting the page on your browser (right-click on the element you want and click on inspect found at the bottom) 👍